# Float: The minimum amount of time that an arbitrated plan must cover for the 
# system to proceed with execution
# Unit: s
min_plan_duration: 6.0

# Float: The nominal amount of time that an arbitrated plan should cover for the 
# system to operate at best performance
# Unit: s
target_plan_duration: 13.0

# Float: The planning frequency for generation for arbitrated plans
# Unit: Hz
planning_frequency: 1.0

# Integer: The width of the search beam to use for arbitrator planning, 1 = 
# greedy search, as it approaches infinity the search approaches breadth-first 
# search
# Unit: N/a
beam_width: 3

# Bool: Use fixed priority cost function over using the cost system for 
# evaluating maneuver plans
# Unit: N/a
use_fixed_costs: true

  # JSON String: Plugin Priorities Map
# It would have been preferable to use a dictionary here but ROS2 Foxy does not support this type of parameter
# Map: The priorities associated with each plugin during the planning 
# process, values will be normalized at runtime and inverted into costs
# Unit: N/a
plugin_priorities: '
  {
 	  "plugin_priorities": 
      {
 			  "platoon_strategic_ihp": 4.0,
        "lci_strategic_plugin": 3.0,
        "sci_strategic_plugin": 2.0,
        "platoon_strategic": 2.0,
        "route_following_plugin": 1.0
      }
  }
  '
